package.json

```diff
@@ -37,7 +37,8 @@
   "dependencies": {
     "fs-extra": ">=3.0.0",
     "sharp": ">=0.22.0",
-    "svg2vectordrawable": "2.6.15"
+    "svg2vectordrawable": "2.6.15",
+    "svgo": "1.2.2"
```

Comment:
Fix `svgo` version due to new incompatible optimisation. See https://github.com/Ashung/svg2vectordrawable/issues/33